lib/membrane_rtc_engine/endpoints/hls_endpoint.ex

```diff
@@ -231,6 +256,24 @@ if Enum.all?(
       {:ok, state}
     end
 
+    @impl true
+    def handle_prepared_to_playing(context, state) do
+      directory = state.output_directory
+      # remove directory if it already exists
+      File.rm_rf(directory)
+      File.mkdir_p!(directory)
+
+      hls_sink_spec = get_hls_sink_spec(state, %{stream_id: :error}, directory)
```

Comment:
I think that `stream_id` `:error` can be a little bit confusing.

Reply:
I will set it to nil to remember to fix version without compositor later on.